pkgs/tools/misc/grub/2.0x.nix

```diff
@@ -53,6 +55,13 @@ stdenv.mkDerivation rec {
     ./fix-bash-completion.patch
   ];
 
+  patchPhase = if kbdcompSupport then ''
```

Comment:
Overriding `patchPhase` breaks `patches` field. Use `prePatch` or `postPatch` instead.

Reply:
Updated to use `postPatch` and tested. Thanks for reviewing this :)